pkg/refund/client.go

```diff
@@ -0,0 +1,100 @@
+package refund
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/httpclient"
+)
+
+const (
+	urlBase            = "https://api.mercadopago.com/v1/payments/{id}/"
+	urlRefundWithParam = urlBase + "refunds/{refund_id}"
+	urlRefund          = urlBase + "refunds"
```

Comment:
o que acha de fazer assim?
```suggestion
	urlBase            = "https://api.mercadopago.com/v1/payments/{id}/refunds"
	urlWithParam = urlBase + "/{refund_id}"
```

Reply:
Mesmo acima